pkg/api/v1beta2/dynakube/convert_from.go

```diff
@@ -173,45 +175,38 @@ func fromAppInjectSpec(src v1beta2.AppInjectionSpec) *AppInjectionSpec {
 	return dst
 }
 
-func (dst *DynaKube) fromMetadataEnrichment(src *v1beta2.DynaKube) {
+func (dst *DynaKube) fromMetadataEnrichment(src *v1beta3.DynaKube) {
 	dst.Spec.MetadataEnrichment.Enabled = src.Spec.MetadataEnrichment.Enabled
 	dst.Spec.MetadataEnrichment.NamespaceSelector = src.Spec.MetadataEnrichment.NamespaceSelector
 }
 
-func (dst *DynaKube) fromExtensions(src *v1beta2.DynaKube) error {
-	delete(dst.Annotations, api.AnnotationDynatraceExtensions)
-	delete(dst.Annotations, api.AnnotationDynatraceOpenTelemetryCollector)
-	delete(dst.Annotations, api.AnnotationDynatraceextEnsionExecutionController)
-
-	e, ok := src.Annotations[api.AnnotationDynatraceExtensions]
-	if ok {
-		es := ExtensionsSpec{}
-		if err := json.Unmarshal([]byte(e), &es); err != nil {
-			return err
-		}
+func (dst *DynaKube) fromExtensions(src *v1beta3.DynaKube) error {
+	e := src.Spec.Extensions
```

Comment:
if `v1beta3` is the stored version then I don't think we need to do this, as no data will be lost.

Reply:
what about when someone apply v1beta3 and we release new version with new stored version? If I remove it will gone?

Reply:
removed. :)